examples/niscope/graph-measurement-betterproto.py

```diff
@@ -119,7 +119,7 @@ async def ConfigureGrpcScope(scope_service: niscope_grpc.NiScopeStub, channel, v
     set_trigger_result = await scope_service.set_attribute_vi_int32(
         vi = vi,
         attribute_id = niscope_grpc.NiScopeAttributes.NISCOPE_ATTRIBUTE_TRIGGER_TYPE,
-        value = niscope_grpc.NiScopeInt32AttributeValues.NISCOPE_INT32_TRIGGER_TYPE_VAL_EDGE_TRIGGER
```

Comment:
> The problem is that zero-value messages in a oneof group take priority based on order and overwrite other set values. 

I think I understand this issue, but the way it's worded is confusing to me.

The spec'd implementation of `oneof` in protobuf is that only one can be set. If you set multiple fields of the `oneof` the "last one wins". It sounds like `betterproto` will set all fields on the `oneof` even if you didn't pass anything in for an argument. So, before your change, `value_raw` would be assigned of default zero-value after `value` is set and that would clear `value`.

Is that right?

Reply:
Yes, that seems to be the behavior. This behavior (bug) is also documented at top of this example and in our wiki under 2nd bullet here: [creating-a-python-grpc-client](https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client#creating-a-python-grpc-client).

 I agree that the wording is confusing.